velox/dwio/parquet/reader/ParquetData.h

```diff
@@ -35,14 +35,18 @@ class ParquetParams : public dwio::common::FormatParams {
   ParquetParams(
       memory::MemoryPool& pool,
       dwio::common::ColumnReaderStatistics& stats,
-      const FileMetaDataPtr metaData)
-      : FormatParams(pool, stats), metaData_(metaData) {}
+      const FileMetaDataPtr metaData,
```

Comment:
Hi @liujiayi771, there is an internal linter complaint that `metaData` is a const non-primitive object passed by value. Is this intentional? I wonder whether it should be passed by const reference or we should std::move it to metaData_. Please let me know if we're intentionally passing it by value and I should ignore that linter complaint.

Also, it would be helpful if you could rebase this PR on top of the latest main. Thank you!

Reply:
Hi @kagamiori, this PR did not make modifications to the `metaData`. This internal linter complaint may have been there all along. If possible, I can address it in a following PR. I will make sure to confirm whether passing by reference is feasible.

Reply:
Sounds good. I'll ignore that complaint for now. Thanks!